openupgrade_scripts/scripts/hr_recruitment/17.0.1.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,47 @@
+---Models in module 'hr_recruitment'---
+---Fields in module 'hr_recruitment'---
+hr_recruitment / hr.applicant             / activity_user_id (many2one)   : not related anymore
+hr_recruitment / hr.applicant             / activity_user_id (many2one)   : now a function
+# NOTHING TO DO: not stored field
+
+hr_recruitment / hr.applicant             / applicant_properties (properties): NEW hasdefault: compute
+# NOTHING TO DO: new properties field
+
+hr_recruitment / hr.applicant             / application_status (selection): selection_keys is now '['archived', 'hired', 'ongoing', 'refused']' ('['hired', 'ongoing', 'refused']')
+# NOTHING TO DO: extra key added in selection
+
+hr_recruitment / hr.applicant             / email_normalized (char)       : NEW isfunction: function, stored
+hr_recruitment / hr.applicant             / message_bounce (integer)      : NEW hasdefault: default
+hr_recruitment / hr.applicant             / partner_mobile_sanitized (char): NEW isfunction: function, stored
+hr_recruitment / hr.applicant             / partner_phone_sanitized (char): NEW isfunction: function, stored
+hr_recruitment / hr.applicant             / phone_mobile_search (char)    : NEW
+hr_recruitment / hr.applicant             / phone_sanitized (char)        : NEW isfunction: function, stored
```

Comment:
too many functions. shouldn't we try to fill this new columns by query? 🤔

Reply:
I looked into these ones and they are calling mixin python functions to sanitize / format email and phone wihch for phone depends on localization for instance, so I think it makes more sense to let Odoo compute these fields with Python functions rather than trying to do the same in SQL to reduce migration time !